src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -301,23 +300,28 @@ public boolean isEnabled() {
 	    return true;
     }
     public void resetEncoders() {
+
         frontLeftEncoder.setPosition(0);
         rearLeftEncoder.setPosition(0);
         frontRightEncoder.setPosition(0);
         rearRightEncoder.setPosition(0);
+        
+        //This approach is simpler than referencing each individual encoder.
+        //normally that's not ok, but in this case, it is safe because we dont expose the individual encoders anywhere
+    	referenceAvgPosition = getAveragePosition();        
     }
 
     /**
      *
      * @return average motor revolutions for the 4 motors
      */
-    public double getAveragePosition() {
+    private double getAveragePosition() {
         double position = 0;
```

Comment:
GetAveragePosition is used by several of the DriveDistance commands.  Are you sure this can be declared as private?

Reply:
I take it back.  we ask for the averagePosition in Meters not motor revolutions.